DOC-1824: [Partner Ops] User binding errors (#688)
swan-io/docs: docs/topics/operations/guide-troubleshoot-binding-errors.mdx

---
title: Troubleshoot user binding errors
---

# Troubleshoot user binding errors

This guide helps partner support teams identify, resolve, and prevent user binding errors.

For technical reference and API integration guides, see [Fix a user binding error](/topics/accounts/memberships/guide-fix-binding-error) and [Update a membership](/topics/accounts/memberships/guide-update) on docs.swan.io.

:::info Key takeaways

- A binding error occurs when the details you provided for an invited account member don't match their sign-up details.
- You can spot binding errors from your Dashboard. To fix them, direct an account member who has the `canManageAccountMembership` permission, usually the legal representative, to resolve the error in Web Banking.

> - Users can often fix two error types (`emailVerifiedMatchError` and `idVerifiedMatchError`) themselves. If the email doesn't match or the identification link has expired, you need to raise a support request in your Dashboard.

Reviewer: ```suggestion
- Users can often fix two error types (`emailVerifiedMatchError` and `idVerifiedMatchError`) themselves. If the email doesn't match or the identification link has expired, raise a support request in your Dashboard.
```

Drops "you need to" so the line is a plain instruction rather than an exclusivity claim — accurate whichever way DOC-1921 resolves, unchanged in meaning for this audience.

- The most important prevention tip: check spelling against the user's identity document, not their preferred name.

:::

---

## Understand binding errors

A binding error occurs when the information you provided when inviting an account member doesn't match the information they provided during sign-up. Until the error is resolved, the member can't perform sensitive operations. What they can see depends on your integration: some members may still view account and card information; others may see nothing at all.

Binding errors only occur during initial binding. Updating personal details on an `Enabled` membership doesn't cause a binding error.

| Scenario | Error code | Example | Resolved by |
| --- | --- | --- | --- |

> | Name spelling mismatch | `firstNameMatchError`, `lastNameMatchError` | You entered "Poole" but the ID says "Pool" | Legal representative |

Reviewer: ```suggestion
| Name spelling mismatch | `firstNameMatchError`, `lastNameMatchError` | You entered "Dupond" but the ID says "Dupont" | Legal representative |
```
Nit: approved example names from the style guide (Henri Dupont), same one-letter mismatch.

| Married vs. birth name | `lastNameMatchError` | You used the married name but the ID shows birth name | Legal representative |

> | Phone number typo | `mobilePhoneMatchError` | A digit was wrong when membership was created | Legal representative |

Reviewer: ```suggestion
| Phone number typo | `phoneNumberMatchError` | A digit was wrong when membership was created | Legal representative |
```
The field on `AccountMembershipBindingUserErrorStatusInfo` is `phoneNumberMatchError` in the current schema — `mobilePhoneMatchError` no longer exists. The API guide and partial on main carry the old name too; follow-up ticket flagged in the general comment.

| Date format swap | `birthDateMatchError` | Day and month swapped (03/06 vs. 06/03) | Legal representative |

> | Different email | `emailVerifiedMatchError` | Invited with work email; user signed up with personal email | Swan Support |

Reviewer: Question: the API guide's causes table says an email mismatch is fixed by updating `email` with `updateAccountMembership` (the input still accepts `email`), but this row routes it to Swan Support only. If CX confirmed support is the real-world path, we should open a follow-up to align the API guide so the two pages don't contradict each other. If the API path still works, consider noting it here for API-integrated partners. The key takeaway on R15 phrases it the same way, so both lines follow whatever the answer is.

Reviewer 2: So would you suggest it be like this: Swan support or API Integration `updateAccountMembership` ?

Reviewer: Only if the API path is confirmed to actually clear the error, as far as I can tell, we never got a definitive answer on that.

**What I did find**: the API guide contradicts *itself* on this point. Its "Who can fix binding errors" section lists `emailVerifiedMatchError` as user-resolvable only and excludes it from the mutation-fixable error types, while its causes table (and the example mutation) say to update `email` via `updateAccountMembership`. So I'll open a follow-up ticket to align that page regardless of what we decide here.

CX or PM should confirm.

Then:
- **If it clears:** your wording, tightened for the table 
  > Swan Support (API partners: [`updateAccountMembership`](/topics/accounts/memberships/guide-fix-binding-error))
  
  with the same pointer mirrored in the key takeaway:
  > …raise a support request in your Dashboard (API-integrated partners can fix an email mismatch with `updateAccountMembership`).
- **If it doesn't clear:** both lines stay exactly as they are, and the correction lands only on the API guide side.

Reviewer: Proposal to unblock: ship this row as-is. Routing an email mismatch to Swan Support is the correct action for this article's audience either way. The page is Dashboard-scoped, and raising a support request is available and safe today regardless of how DOC-1921 resolves.

The only phrasings that aren't safe under both outcomes are (a) asserting the API path works (unverified) and (b) asserting Support is the *only* path (also unverified). The draft has one instance of (b)("you need to raise" on R15) so I've left a suggestion there dropping the exclusivity. The table cell stays `Swan Support`.

If DOC-1921 confirms the API path, we add the parenthetical pointer in a one-line follow-up; if it doesn't, this page is already right. (I'd hold off pointing API partners to the fix-binding-error guide from this row until DOC-1921 lands, since that's the page being corrected.) Either way, publication no longer waits on the verification.

@jthies-star good to apply the R15 suggestion and re-request review?

Reviewer 2: Sounds good to me!

| Email not verified | `emailVerifiedMatchError` | Email matches but user hasn't clicked the verification link | User — see [below](#help-users-fix-their-own-binding-errors) |

> | Identification not completed | `idVerifiedMatchError` | User hasn't completed the required identification | User — see [below](#help-users-fix-their-own-binding-errors) |

Reviewer 2: The linking needs to be adjusted to GitHub, I think.

> | Identification not completed | `idVerifiedMatchError` | User hasn't completed the required identification | User — see [below](#help-users-fix-their-own-binding-errors) |

Reviewer: ```suggestion
| Email not verified | `emailVerifiedMatchError` | Email matches but user hasn't clicked the verification link | User, see [how users fix it](#help-users-fix-their-own-binding-errors) |
| Identification not completed | `idVerifiedMatchError` | User hasn't completed the required identification | User, see [how users fix it](#help-users-fix-their-own-binding-errors) |
```
Style guide: avoid directional markers like "below", descriptive link text instead.

---

## Spot a binding error

1. Go to **Dashboard** > **Data** > **Account memberships**.
1. Filter for membership status `BindingUserError`.
1. Select the affected user.

Your Dashboard shows all mismatched information. The Dashboard doesn't let you correct membership data directly. Your user may be able to correct the mismatch themselves, depending on your integration. Check the integration setup before moving on.

:::info Can't find it?

> If you don't find a `BindingUserError` status, the user may not have completed the onboarding flow yet. This isn't a binding error. Raise a support request in your Dashboard for guidance.

Reviewer: ```suggestion
If you don't find a `BindingUserError` status, the user may not have accepted the invitation and completed sign-up yet. This isn't a binding error. Raise a support request in your Dashboard for guidance.
```
"Onboarding" means creating an account holder in Swan terminology. Members accept an invitation and sign up — the membership sits at `ConsentPending` or `InvitationSent` before binding.

Make sure they're using the same email they originally signed up with.

:::caution Wrong email address verified

If the user verifies a different email address than the one shown on their membership, raise a support request in your Dashboard. Include the correct email address and phone number.

:::

### Identification not completed (`idVerifiedMatchError`)

Tell your user that they need to complete their identity verification. Ask them to open the link they received by email or SMS to start.

If the user's identification link has expired or is no longer accessible, raise a support request in your Dashboard to issue a new one. Include the account membership ID in your request.

---

## Prevent binding errors

1. **Check spelling against the identity document.** Use the exact name on the ID, not their preferred name.
1. **Use the user's birth name**, not their married name.
1. **Confirm the phone number format.** Include the country code (for example, +33 for France, +34 for Spain, +49 for Germany, +39 for Italy, +351 for Portugal, +31 for the Netherlands).
1. **Verify the email address.** Confirm which email the user will sign up with.

> 1. **Don't swap day and month** in birth dates.

Reviewer: ```suggestion
1. **Check spelling against the identity document.** Use the exact name on the ID, not their preferred name.
2. **Use the user's birth name**, not their married name.
3. **Confirm the phone number format.** Include the country code (for example, +33 for France or +49 for Germany).
4. **Verify the email address.** Confirm which email the user will sign up with.
5. **Don't swap day and month** in birth dates.
```
Two country-code examples make the point — members can be located anywhere, so the long list of Swan account countries reads like a restriction.

---

## Contact Swan Support

Raise a [Swan Support request](https://partner-support.swan.io/hc/en-gb/requests/new) if:

> - The error persists after correcting the mismatched information, and the user has consented.

Reviewer: ```suggestion
- The error persists after the legal representative corrected the mismatched information and gave consent.
```
"The user has consented" is ambiguous — in the Web Banking flow (this page's fix path), the resolver consents on their device.

- The membership is stuck in `BindingUserError` after the user has completed identification and email verification.
- You need to resolve a binding error for the legal representative's membership.

Include in your request:

- Account ID and account membership ID

> - The specific error type(s)

Reviewer: ```suggestion
- The specific error types
```
Nit: plain English; several error types can apply at once, so plural is safe.

- Resolution steps already taken

---

## Share these articles with your users

Share these Support Center articles with users who have questions about account membership:

- [Resolve a user binding error](https://support.swan.io/hc/en-150/articles/20431032028829-Resolve-a-user-binding-error)
- [Add a member to your account](https://support.swan.io/hc/en-150/articles/17648698750877-Add-a-member-to-your-account)

> - [Identification process overview](https://support.swan.io/hc/en-150/articles/22865683055133-Identification-process-overview)

Reviewer: ```suggestion
- [Resolve a user binding error](https://support.swan.io/hc/en-150/articles/20431032028829-Resolve-a-user-binding-error): fixing the error in Web Banking, step by step.
- [Add a member to your account](https://support.swan.io/hc/en-150/articles/17648698750877-Add-a-member-to-your-account): inviting a new member with correct details.
- [Identification process overview](https://support.swan.io/hc/en-150/articles/22865683055133-Identification-process-overview): what to expect during identification.
```
Nit: the transfers guide adds a one-line "when to share it" description per article — helps ops pick the right one fast.

:::tip Help center content

You can also use these articles as a starting point to create your own FAQ or help center content, tailored to your product's terminology and your users' specific context.

> :::

Reviewer: ```suggestion
:::

```
Missing newline at end of file.

> :::

Reviewer: ```suggestion
:::

```
